src/pages/register-a-beacon/beacon-information.tsx

```diff
@@ -105,7 +105,8 @@ const ManufacturerSerialNumberInput: FunctionComponent<FormInputProps> = ({
     >
       <Image
         src="/assets/mca_images/beacon_serial_number.png"
-        alt="Where to find your beacon's manufacturer serial number"
+        alt="This image illustrates what a beacon's manufacturer serial number looks like
```

Comment:
Nice.  It's worth having a look to see if we have missed any other alt text.  Top tip: you should be able to find all occurrences using: `git grep "alt="`

See git grep docs: https://git-scm.com/docs/git-grep

Reply:
So the remaining two are for logos: ![image](https://user-images.githubusercontent.com/88549170/131853292-734e876b-62eb-45c8-8dcb-1019e76e22b8.png)

Do you think these could be more informative?